src/tasks/phase2/database_connection.py

```diff
@@ -0,0 +1,161 @@
+import psycopg2
+import numpy as np
+import pickle
+'''
+    This class has database connection class.
+    This class have get_db_connection method which will return database cursor.
+
+'''
+
+
+class DatabaseConnection:
+
+    label_type_map = {
+        "aspect":"aspectofhand",
+        "gender":"gender",
+        "accessories":"accessories"
+    }
+
+    def get_db_connection(self):
+        psycopg2.extensions.register_type(psycopg2.extensions.UNICODE)
+        connection = psycopg2.connect(host='localhost', port='5432', user='postgres',
+                                password='postgres', dbname='Hands_db')
+        return connection
+
+    def create_feature_model_table(self, tablename):
+        connection = self.get_db_connection()
+        cursor = connection.cursor()
+        cursor.execute("""DROP Table IF EXISTS {};""".format(tablename))
+        # Create metadata table
+        cursor.execute("""CREATE TABLE IF NOT EXISTS {0}(
+                                imageName TEXT NOT NULL,
+                                features BYTEA NOT NULL,
+                                PRIMARY KEY (imageName)
+                                );
+                                """.format(tablename))
+        connection.commit()
+
+    def insert_feature_data(self, tablename, feature_vectors):
+        connection = self.get_db_connection()
+        cursor = connection.cursor()
+        cursor.executemany("""INSERT INTO {0}(imageName,features) VALUES (%(imageName)s, %(features)s)""".
+                           format(tablename), feature_vectors)
+        connection.commit()
+
+    def get_feature_data_for_image(self, tablename, imageName):
+        """
+        Returns the feature vector for an image for a model
+        Output Shape: [feature vector]
+        """
+        connection = self.get_db_connection()
+        cursor = connection.cursor()
+        cursor.execute("SELECT features from {0} where imagename = '{1}'".format(tablename, imageName))
+        result_row = cursor.fetchone()
+        return np.array(pickle.loads(result_row[0]))
+
+    def get_subject_ids_in_db(self, tablename):
```

Comment:
Do we need subject related functions anymore?

Reply:
Don't think we need subject related functions anymore